lectures/_static/lecture_specific/orth_proj/orth_proj_thm2.tex

```diff
@@ -14,11 +16,12 @@
     \coordinate (Py') at (-0.28,-0.14);
     \draw[axis] (-0.5,0)  -- (0.9,0) node(xline)[right] {};
     \draw[axis] (0,-0.3) -- (0,0.7) node(yline)[above] {};
+    \draw[important line, thick]  (Z1) -- (O);
+    \draw[important line, thick]  (Py) -- (Z2) node[right] {$S$};
     \draw[important line,blue,thick, ->]  (O) -- (Py) node[anchor = north west, text width=2em] {$P y$};
     \draw[important line,blue, ->]  (O) -- (y') node[left] {$y'$};
-    \draw[important line, thick]  (Z1) -- (O) node[right] {};
-    \draw[important line, thick]  (Py) -- (Z2) node[right] {$S$};
     \draw[important line, blue,->]  (O) -- (y) node[right] {$y$};
+    \draw[important line,blue,thick, ->]  (O) -- (Py');
```

Comment:
I built this but it still generates the same old figure, so I updated this line to

```
    \draw[important line, blue,->]  (O) -- (Py') node[anchor = north west, text width=5em] {$P y'$};
```

following the previous lines which gives the blue arrow!